users/templates/registration/login.html

```diff
@@ -0,0 +1,72 @@
+{% extends "base_generic.html" %}
+{% load static %}
+
+{% block css %}
+    <!-- Favicon-->
+    <link rel="shortcut icon" href="{% static 'img/fav.png' %}">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,200,400,300,500,600,700" rel="stylesheet">
+    <!--CSS============================================= -->
+    <link rel="stylesheet" href="{% static 'css/linearicons.css' %}">
+    <link rel="stylesheet" href="{% static 'css/font-awesome.min.css' %}">
+    <link rel="stylesheet" href="{% static 'css/bootstrap.css' %}">
+    <link rel="stylesheet" href="{% static 'css/magnific-popup.css' %}">
+    <link rel="stylesheet" href="{% static 'css/nice-select.css' %}">
+    <link rel="stylesheet" href="{% static 'css/animate.min.css' %}">
+    <link rel="stylesheet" href="{% static 'css/owl.carousel.css' %}">
+    <link rel="stylesheet" href="{% static 'css/main.css' %}">
+{% endblock %}
+
+{% block banner %}
+    <section class="banner-area" style="height: 800px">
+        <div class="container">
+            <div class="row fullscreen d-flex align-items-center justify-content-start">
+                <div class="banner-content col-lg-7">
+                    <h5 class="text-white text-uppercase">Author: Travor James</h5>
+                    <h1 class="text-uppercase">
+                        New Adventure
+                    </h1>
+                    <p class="text-white pt-20 pb-20">
+                        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod temp <br> or incididunt ut labore et dolore magna aliqua. Ut enim ad minim.
+                    </p>
+                    <a href="#" class="primary-btn text-uppercase">Review now</a>
+                </div>
+                <div class="col-lg-5 banner-right">
+                    <img class="img-fluid" src="{%static 'img/header-img.png' %}" alt="">
+                </div>
+            </div>
+        </div>
+    </section>
+{% endblock %}
+
+{% load crispy_forms_tags %}
+{% block content %}
+<h3 style="text-align: center">Login</h3>
+<form method="post" class="formsWidth pb-20 pt-20">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <br>
+    <button type="submit" class="btn btn-success">Login</button>
+    <span><a style="float: right" href="{% url 'password_reset' %}">Forgot your password?</a></span>
```

Comment:
tìm hiểu i18n trong django để quốc tế hóa các chuỗi hiên thị lên view cho người dùng nha em
yêu cầu translate toàn bộ các chuỗi ở những phần UI đã hoàn thành, những chỗ fake data thì bỏ qua